### PR TITLE
blogpost about protractor in ionic

### PR DESCRIPTION
**Pull request by yannickvergeylenord**
wrote blogpost about protractor in ionic, added 2 images for the content of the post and 1 as cover

### CODE REVIEW

**Comment by tomverelst** [comment]
Please put each sentence, or part of a sentence on different line in the Markdown file. Easier for versioning and giving feedback on sentences. Thanks!

**Comment by Turbots** [comment]
@MrJean can remain assigned to check the content of the technical stuff a bit better :yum:

File: _posts/2016-04-07-Ionic-Protractor.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+ionic serve
+{% endhighlight %}
+
+> It's worth noting that Ionic has live reload built in **by default**.
+> So any changes will be immediately
+reflected in the browser.
+
+To view the application using the iOS and Android styling applied you can use the following command:
+
+{% highlight bash %} ionic serve -lab {% endhighlight %}
```
Comment (MrJean) [inline _posts/2016-04-07-Ionic-Protractor.md]: ionic serve **--**lab

**Comment by MrJean** [comment]
@yannickvergeylenord I re-read the post and it's OK for me but would it be possible to add a conclusion to your post? Something that gives a summary of the post and also motivates readers to read the whole post.

**Pull request merged**